pkg/ddevapp/db_test.go

```diff
@@ -36,6 +36,7 @@ func TestDBTypeVersionFromString(t *testing.T) {
 		"mariadb_10.7":  "mariadb:10.7",
 		"mariadb_10.11": "mariadb:10.11",
 		"mariadb_11.4":  "mariadb:11.4",
+		"mariadb_11.8":  "mariadb:11.8",
```

Comment:
initially at least this will be `mariadb:11.8-rc`

Reply:
We only do major, so it will be 11.8. I imagine they'll have that as the master tag when they get it pushed.